Controllers/PrayerRequestsController.cs

```diff
@@ -0,0 +1,81 @@
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using WebApi.Models;
+using WebApi.Data;
+namespace WebApi.Controllers
+{
+    [Route("api/[Controller]")]
+    [ApiController]
+    public class PrayerRequestsController : ControllerBase
+    {
+        private readonly ApiContext _context;
+
+        public PrayerRequestsController(ApiContext context)
+        {
+           _context = context;
+        }
+      
+      //Create and Edit prayer request
+      //POST: api/newRequest
```

Comment:
it can be called "new" because the actual call will be api/prayerRequests/new  every function you create here will follow the api/prayerRequests/[fucntionpath].